app/viewmodels/application/GoodsDetailsSummary.scala

```diff
@@ -53,19 +58,29 @@ object GoodsDetailsSummary {
         )
     }
 
-    val confidentialInformationRow = goodsDetails.confidentialInformation.map {
-      info =>
-        SummaryListRowViewModel(
-          key = "confidentialInformation.checkYourAnswersLabel",
-          value = ValueViewModel(info)
-        )
-    }
+    val confidentialInformationRow: Option[SummaryListRow] =
+      goodsDetails.confidentialInformation.map {
+        info =>
+          SummaryListRowViewModel(
+            key = "confidentialInformation.checkYourAnswersLabel",
+            value = ValueViewModel(info)
+          )
+      }
+
+    val attachmentRowContent = attachments
```

Comment:
I believe this could be covered with an additional case in GoodsDetailsSummarySpec

Reply:
Added